orchestrator/tests/nl2sql_eval/harness.py

```
@@ -0,0 +1,147 @@
+        if qr.correct:
+            report.correct += 1
+        report.results.append(qr)
+
+    conn.close()
+    return report
+
+
+def check_goldens() -> EvalReport:
+    """Always-on integrity: every golden SQL must execute on the seed (an
+    identity eval where the candidate IS the golden — accuracy must be 1.0)."""
+    return evaluate(lambda question, schema, _q=load_questions(): next(
```

Ruff (0.15.15)

[warning] Do not perform function call `load_questions` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)